.circleci/config.yml

```diff
@@ -116,7 +116,7 @@ jobs:
       - image: circleci/python:2.7
     working_directory: ~/st2
     environment:
-      - DISTROS: "trusty xenial bionic el6 el7"
+      - DISTROS: "trusty xenial bionic el6 el7 el8"
```

Comment:
Need to adjust parallelism as well, otherwise there are only `5` containers in CircleCI build, see https://circleci.com/gh/StackStorm/st2/10882